src/caformat-util.c

```diff
@@ -112,6 +112,7 @@ static const struct {
         { "fat-attrs",        CA_FORMAT_WITH_FAT_ATTRS        },
         { "privileged",       CA_FORMAT_WITH_PRIVILEGED       },
         { "fuse",             CA_FORMAT_WITH_FUSE             },
+        { "all",              CA_FORMAT_WITH_MASK             },
```

Comment:
hmm, so, --with=all only makes limited sense since many of the flags actually contradict each other. i.e. the various time granularity bits can't be applied together, you can only select a single granularity after all...

--with=best is actually what --with=all is supposed to be: all features turned on, and if two features collide the "better" one is picked...

Reply:
hmm, so i read #81 now, and i see the usecase for this. It's --without=all where this is useful, not --with=all. Maybe this patch is OK then, but we should probably change the --help text for this specific entry, and only suggest people to use "--without=all", and never mention "--with=all".

(--with=all will work too btw, as casync will clean up redundant bits automatically, but it's still confusing, and --with=best is much clearer I think)

Reply:
I updated the docs and `--help` output. `--with=all` is still possible, but it's explicitly discouraged in the man page and not advertised elsewhere.